README.md

```diff
@@ -1,12 +1,12 @@
 [ ![Codeship Status for databraid-dashboard/sheet-api](https://app.codeship.com/projects/ea3e8280-7fa4-0135-2d31-72fb077510aa/status?branch=master)](https://app.codeship.com/projects/246443)
 
-# Databraid API Starter Repo Without Postgres
+# Databraid Sheet API
 
 Use this repo as a starter repo for developing API servers without Postgres
```

Comment:
@kurtishouser I know this isn't your line of code here, but it needs to be deleted (it is left over from the starter repo).